docs/source/defining-shortcuts.md

```diff
@@ -144,6 +153,14 @@ shortcut.
   communication mechanism). Note that setting `CFBundleTypeRole` will make the wrapper blip in the
   dock when the URL is opened. If you don't want that, do not set it.
 
+:::{note}
```

Comment:
Why is this note included twice so close to the other one? Mistake?

Reply:
Ah, it's the same note for two related features: associating file extensions and URL protocols. In macOS, these are handled by the same system, but it's not the same in other platforms. So just to make sure folks do see it if they are only handling one of the cases.

Reply:
Okay, sounds good to me.